models/dbfs/dbfile.go

```diff
@@ -193,10 +195,26 @@ func (f *file) Close() error {
 	return nil
 }
 
+func (f *file) Stat() (os.FileInfo, error) {
+	if f.metaID == 0 {
+		return nil, os.ErrInvalid
```

Comment:
Other code returns `os.ErrNotExist` for this case

Reply:
Hmm ... not consistent. This check should never be true because Open would never return a "file" with ID=0.

So I think it's fine